### DISCUSSION TITLE
Add more functions to image previewer

### DISCUSSION THREAD

**Discussion by szsdk** [sxyazi/yazi#1633] [4. Ideas]
Thanks for this great tool. `yazi` is the best in supporting different terminals and protocol from my experience. Viewing some images via ssh is a daily job for me. Is it possible to add some functions like "zoom-in" into to the previewer?

**Comment by sxyazi** [comment]
> Is it possible to add some functions like "zoom-in" into to the previewer?

It's now supported with https://github.com/yazi-rs/plugins/tree/main/zoom.yazi
